lib/index.js

```diff
@@ -182,37 +182,36 @@ Mixpanel.prototype.track = function(track, fn){
 
 Mixpanel.prototype.screen =
 Mixpanel.prototype.page = function(msg, fn){
-  var batch = new Batch;
   var category = msg.category();
-  var name = msg.fullName();
+  var name = msg.name();
+  var fullName = msg.fullName(); // includes both category and name
   var self = this;
+  var sent = false; // Prevents sending more than one call per page/screen
```

Comment:
We don't need this flag anymore. Just return after you make the first call to `send`.